src/pages/Cart/Cart.js

```diff
@@ -1,242 +1,154 @@
-                        >
-                          <i className="fas fa-plus cart-icons"></i>
-                        </button>
-                      </div>
-                      
-                      <div className="card-footer">
-                      <button
-                        className="btn btn-text-icon-primary grid-horizontal-icons"
-                      >
-                        Remove from Cart
-                      </button>
-                     <Link to="/wishlist"> <button
-                      className="btn btn-text-icon-primary grid-cards-icons gridd-icons"
-                    >
-                      Move to Wishlist
-                    </button></Link>
-                      </div>
+                      <img
+                        src={product.img}
+                        alt={product.alt}
+                        className="grid-col-img cards-horizontal-img"
+                      />
                     </div>
-                    <span className="horizontal-tag">New</span>
-                    <button type="button" className="btn-dismissal">
-                <i className="fa-solid fa-xmark card-btn-dismissal"></i>
-              </button>            
-              </div>
-            </li>
 
-            <li className="cart-main">
-                <div className="cardss cards-horizontals">
-                    <div className="img-container">
-                    <img
-                  src="https://github.com/ranitasahaa1230/Trekkart-Store/blob/d51db72677ff63c4de681f8b249d14fcab32f3ac/components/images/luggage2.jpg?raw=true"
-                  alt="hiking"
-                  className="grid-col-img cards-horizontal-img"
-                />
-              </div>
-                     
                     <div className="card-content padding-s">
-                      <h3 className="card-horizontal-grid">
-                        Stylish Bags
-                      </h3>
-      
+                      <h3 className="card-horizontal-grid">{product.name}</h3>
+
                       <div className="sm-gap md-gap">
-                        <span className="txt-bold"> Rs.3899</span>
-                        <span className="txt-crossed-off">Rs.2999</span>
-                        <span className="txt-high-light">10% Off</span>
+                        <span className="txt-bold">
+                          ₹
+                          {new Intl.NumberFormat("en-IN").format(
+                            product.newPrice
+                          )}
+                        </span>{" "}
+                        <span className="txt-crossed-off">
+                          ₹
+                          {new Intl.NumberFormat("en-IN").format(
+                            product.originalPrice
+                          )}
+                        </span>{" "}
+                        <span className="txt-high-light">
+                          {calcPercentage(
+                            product.newPrice,
```

Comment:
instead of writing product.something, you can destructure product and just use the property, this was suggested by tanay as well.

Reply:
Sure, will do it